gems/pending/appliance_console.rb

```diff
@@ -193,11 +231,21 @@ module ApplianceConsole
         if agree("Apply static network configuration? (Y/N)")
           say("\nApplying static network configuration...")
 
-          Env['STATIC'] = new_ip, new_mask, new_gw, new_dns1, new_dns2
+          begin
+            eth0.address = new_ip
+            eth0.netmask = new_mask
+            eth0.gateway = new_gw
```

Comment:
I'm thinking LinuxAdmin should have both.  Individual accessors for when you need that granularity, and helper methods to handle the 80% case where you have to set everything.